src/scripts/Battle.ts

```diff
@@ -26,9 +26,9 @@ class Battle {
      */
     public static pokemonAttack() {
         // TODO: figure out a better way of handling this
-        // Limit pokemon attack speed, Only allow 1 attack per 900ms
+        // Limit pokemon attack speed, Only allow 1 attack per 750ms
         const now = Date.now();
-        if (this.lastPokemonAttack > now - 900) {
+        if (this.lastPokemonAttack > now - 750) {
```

Comment:
Why are we decreasing this?

Reply:
"Also buffed attack speed by a tiny bit to compensate"

Reply:
This doesn't actually increase the pokemon attack speed though, It just allows people to cheat pokemon attack faster

Reply:
I always forget the -900 in Battle.ts doesn't actually control the speed

Reply:
Reverted